woocommerce/woocommerce
# [Product blocks editor]: Add help text to Shipping toggle

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





The changes introduced by this PR show a help section in the shipping section of the Product/Variation.

Closes https://github.com/woocommerce/woocommerce/issues/41509



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to the product editor
2. Ensure to create a product/variations 
3. Go to the Shipping section

Before | After
-------|------
<img width="691" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/baee90a9-e795-4b19-9f71-3b1c4b50759a"> | <img width="693" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/3f75a26b-9f23-43d4-9753-2dfbcb82d715">


https://github.com/woocommerce/woocommerce/assets/77539/83b30963-50da-4cbd-9061-44811d847093






### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
